static/js/browser2/statvar_group_node.tsx

```diff
@@ -53,7 +53,7 @@ interface StatVarGroupNodePropType {
 }
 
 interface StatVarGroupNodeStateType {
-  isRendered: boolean;
+  toggledOpen: boolean;
```

Comment:
add a comment here? specifically how this relates to the prop `open`. maybe the prop should be renamed to something like `startsOpened`

Reply:
done